app/dao/account.go

```diff
@@ -37,3 +37,13 @@ func (r *account) FindByUsername(ctx context.Context, username string) (*object.
 
 	return entity, nil
 }
+
+// AddAccount : ユーザを作成
+func (r *account) AddAccount(ctx context.Context, a object.Account) error {
+	_, err := r.db.ExecContext(ctx, "INSERT INTO account (username, password_hash) VALUES (?, ?)", a.Username, a.PasswordHash)
+	if err != nil {
+		return fmt.Errorf("%w", err)
+	}
```

Comment:
返り値が err のみの場合、短縮系でハンドリングが書けそうです。

```suggestion
	if _, err := r.db.ExecContext(ctx, "INSERT INTO account (username, password_hash) VALUES (?, ?)", a.Username, a.PasswordHash); err != nil {
		return fmt.Errorf("%w", err)
	}
```

このように書くことで、`err` のスコープを if内に閉じることができます。下記の例のようにエラーのハンドリングが複数ある場合にすでに宣言されたかどうかを気にする必要がなくなるためおすすめです

```go
if err := hoge(); err != nil {
	fmt.Println(err)
}

if err := fuga(); err != nil {
	fmt.Println(err)
}
```